markdown/chapters/meandeck/interaction.md

```diff
@@ -354,19 +372,27 @@ serviceable pile from them. Even better if your graveyard is small or
 non-existent; you can cast Doomsday and set your library of 0 cards and then
 cast Oracle to win.
 
+There is a special case scenario with {{Opposition Agent}}. If you cast Doomsday
+with an Opposition Agent in play your opponent gets to search for five cards and
+the rest of your deck is exiled. Because your opponent controls the search they
+will get to see your hand and you will not be able to search from the graveyard.
+The only way to really beat a resolved agent is if you already have Oracle
+castable in hand, hopefully using Cavern of Souls otherwise the opponent could
+search for five counterspells to try and stop you.
+
 ## 4. Playing Around Shuffle Effects
 
 Shuffle effects can be one annoyance to face when building a pile that would
 otherwise have perfect ordering to it. There are some examples that you could
-consider needing to play around like Field of Ruin however this is a bit too
-niche to consider is most matches. The primary concern is in fact a card that is
-very prevalent in Legacy; Surgical Extraction.
-
-Upon successfully resolving a Doomsday, assuming no cards like Leyline of the
-Void are present, there will be one card in your graveyard and that will be the
-copy of Doomsday itself. This is a prime target for an opposing Surgical in an
-attempt to mess up the ordering of a Doomsday pile as you try and *go off* in
-one turn. When the surgical is used can depend on the opponent but if they wait
+consider needing to play around like {{Field of Ruin}} or {{Assassin's Trophy}}
```

Comment:
Field forces the search (and therefore shuffle) unlike Trophy and Ghost Quarter. Should that be mentioned?

Reply:
Probably not relevant to mention here, maybe in opposing interactions appendix.